site_forecast_app/models/pvnet/model.py

```diff
@@ -90,19 +84,110 @@ def __init__(
             log.exception("Failed to prepare data sources or load model.")
             log.exception(f"Error: {e}")
 
+    def _get_config(self):
```

Comment:
could you move these back to the same position there were. Otherwise its very hard to tell what has changed?